bazel/rules/score_module/private/seooc.bzl

```diff
@@ -0,0 +1,82 @@
+load("@rules_python//sphinxdocs/private:sphinx_docs_library_info.bzl", "SphinxDocsLibraryInfo")
+
+seooc_artifacts = {
+    "assumptions_of_use": attr.label(
+        providers = [SphinxDocsLibraryInfo],
+        mandatory = True,
+        doc = "Label to a sphinx_docs_library target containing the Assumptions of Use, which define the safety-relevant operating conditions and constraints for the SEooC as required by ISO 26262-10 clause 5.4.4.",
+    ),
+    "component_requirements": attr.label(
+        providers = [SphinxDocsLibraryInfo],
+        mandatory = True,
+        doc = "Label to a sphinx_docs_library target containing the component requirements specification, defining functional and safety requirements as required by ISO 26262-3 clause 7.",
+    ),
+    "architectural_design": attr.label(
+        providers = [SphinxDocsLibraryInfo],
+        mandatory = True,
+        doc = "Label to a sphinx_docs_library target containing the architectural design specification, describing the software architecture and design decisions as required by ISO 26262-6 clause 7.",
+    ),
+    "safety_analysis": attr.label(
+        providers = [SphinxDocsLibraryInfo],
+        mandatory = True,
+        doc = "Label to a sphinx_docs_library target containing the safety analysis, including FMEA, FMEDA, FTA, or other safety analysis results as required by ISO 26262-9 clause 8. Documents hazard analysis and safety measures.",
+    ),
+}
```

Comment:
Having everything in separate SphinxDocsLibraryInfo targets means the generated index file will differ from the original index file. So module docu will look different than seeoc docu of the same module. Correct?